src/pudl/metadata/classes.py

```diff
@@ -624,7 +624,9 @@ def to_pyarrow(self) -> pa.Field:
             name=self.name,
             type=self.to_pyarrow_dtype(),
             nullable=(not self.constraints.required),
-            metadata={"description": self.description},
+            metadata={
+                "description": self.description if self.description is not None else ""
```

Comment:
Ideally, no. We want them all to have descriptions.

Practically, this is a bit of work. In the case of the columns, many of the ones that didn't have descriptions were extremely generic, and had notes about the need to disambiguate them between the many different tables that they show up in, since they don't actually always mean the same thing (so they at least need specific descriptions, and maybe they need to be renamed so that there's closer to a 1:1 mapping between column name and description).

And then there's the EIA-861, which has dozens of tables and hundreds of columns with no descriptions.

We can certainly use Pydantic to force them all to have descriptions (and we probably should) but there's probably a couple of days worth of work to get everything filled in with clear and accurate metadata.  Since we don't actually know what all of these columns are.

Reply:
I'm ok to split this off into its own issue if we're able to catalog all the columns that don't currently have descriptions.

Reply:
I've created #3224 enumerating:
- resources without descriptions
- unused fields
- fields without descriptions